configure.ac

```diff
@@ -236,6 +236,24 @@ if test x$enable_bip70 != xno; then
   AC_MSG_ERROR([BIP70 is no longer supported!])
 fi
 
+AC_ARG_WITH([libmultiprocess],
+  [AS_HELP_STRING([--with-libmultiprocess=yes|no|auto],
+  [Build with libmultiprocess library. (default: detect with pkg-config)])],
```

Comment:
nit: The help string could clear state that default=auto:

"(default is auto, i.e., detect with pkg-config)"

or smth similar.